mission_control/navigator_missions/nav_missions/explore.py

```diff
@@ -1,10 +1,37 @@
 #!/usr/bin/env python
 import txros
-import tf
+from tf import transformations as trns
 import numpy as np
 import navigator_tools
 from twisted.internet import defer
 
 @txros.util.cancellableInlineCallbacks
 def main(navigator):
-    navigator.nh.get_service_cleint("/explorer/get_next_point",)
\ No newline at end of file
+    result = navigator.fetch_result()
+
+    # Get all the objects in the db
+    resp = yield navigator.database_query("All")
+    boat_position = navigator.pose[0]
+    if resp.found:
+        p_to_np = navigator_tools.point_to_numpy
+        obj_dist = lambda obj: np.linalg.norm(p_to_np(obj.position) - boat_position)
+        furthest = max(resp.objects, key=obj_dist)
+
+        yield navigator.move.set_position(p_to_np(furthest.position)).go()
+        
+        # Set that fake marker to false or something to denote we went there
+        yield navigator.database_query(cmd="{}=0, 0".format(furthest.name))
+
+        result.message = "Went to {}.".format(furthest.name)
+
+    else:
+        # Random move if we have no labled points
+        yield navigator.move.yaw_left(np.random.normal() * 6.28).forward(50).go()
```

Comment:
Ohhh I like the way you think! Will do (but not in this PR)

Random number just moves us and let's rng-Jesus guide us. Bounds are hard enforced so points out side of them would get rejected and the mission would just run again. If we enforce bounds at a planner level, that would make us move to the edge (which I plan to do)